app/Http/Controllers/WorkScheduleController.php

```diff
@@ -40,27 +49,70 @@ public function index()
         }
 
         $locations = $this->locationRepository->getByWorkspace(Auth::user()->workspace_id);
+        $seats = Auth::user()->workspace->seats()->pluck('seats.name', 'seats.id');
         $data = $this->workingSchedule->getUserSchedule(Auth::user()->id);
         $dataLocation = $this->workingSchedule->getLocation(Auth::user()->id);
+        $dataSeat = Auth::user()->seats()->pluck('seat_id', 'registration_date');
+
+        if (count($data) == config('site.disallow_register')) {
+            $registered = false;
+        } else {
+            $registered = true;
+        }
 
-        return view('users.registerworkschedules', compact('dates', 'data', 'locations', 'dataLocation'));
+        return view('users.registerworkschedules',
+            compact('dates', 'data', 'locations', 'dataLocation', 'dataSeat', 'seats', 'registered'));
     }
 
     public function registerWork(Request $request)
     {
-        $data = $request->all();
-        foreach ($data['shift'] as $key => $value) {
-            $request->user()->workSchedules()->updateOrCreate(
-                [
-                    'date' => $key,
-                ],
-                [
-                    'location_id' => $data['location'][$key],
-                    'shift' => $value,
-                ]
-            );
+        try {
+            $data = $request->all();
+            foreach ($data['shift'] as $key => $value) {
+                DB::transaction(function () use ($request, $data, $key, $value) {
+                    $request->user()->workSchedules()->updateOrCreate(
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WorkScheduleController.php</br>Closing parenthesis of a multi-line function call must be on a line by itself